playbooks/amc.yml

```diff
@@ -28,6 +28,22 @@
     - role: "{{ appsembler_roles }}/google-fluentd"
     - { role: 'rabbitmq', rabbitmq_ip: '0.0.0.0' }
 
+- name: Deploy edxapp workers
+  hosts: workers
+  become: True
+  become_method: sudo
+  gather_facts: True
+  vars:
+    appsembler_roles: "../../appsembler-roles"
+  roles:
+    - role: sudo
+    - role: aws
+    - mysql_init
```

Comment:
```suggestion
```

I don't think we need this role for `edxapp` workers.

Reply:
@OmarIthawi `mysql_init` puts the SSL certificates and tls connect script in place. It should probably be refactored so that isn't in the same role as creating users/databases, etc. but we probably still need it.

Reply:
@OmarIthawi Yes, as @thraxil pointed, it's needed, if not it fails.